assets/training/finetune_acft_hf_nlp/environments/acpt/context/requirements.txt

```diff
@@ -5,12 +5,14 @@ mltable=={{latest-pypi-version}}
 datasets==2.12.0
 mpi4py==3.1.4
 sentencepiece==0.1.99
-transformers==4.29.1
+transformers==4.31.0
 optimum==1.6.4
-accelerate==0.19.0
+accelerate==0.21.0
 diffusers==0.16.1
 onnxruntime-training==1.15.1
 rouge-score==0.1.2
 sacrebleu==2.2.1
-bitsandbytes==0.38.1
+bitsandbytes==0.41.0
 einops==0.6.1
+peft==0.4.0
+deepspeed==0.9.5
```

Comment:
May be pick the pins from this PR as well to fix vulnerabilities? https://github.com/Azure/azureml-assets/pull/942